README.md

```diff
@@ -60,144 +60,13 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [WebPack boilerplate] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] In your readme file kindly update the Future Features sections with actual features you will add to the project.

- [ ] Kindly update the FAQ section with relevant information or remove the section entirely since it is optional.

Reply:
Done.